code/modules/mining/mine_turfs.dm

```diff
@@ -10,7 +10,7 @@ var/global/list/rockTurfEdgeCache
 	name = "rock"
 	icon = 'icons/turf/mining.dmi'
 	icon_state = "rock_nochance"
-	baseturf = /turf/simulated/floor/plating/asteroid
+	baseturf = /turf/space

```

Comment:
I am PRETTY sure this is a bad idea. it was made to have asteroid turf as its baseturf for a very specific reason, so it can't be spaced.